tests/dummy/app/components/file-tree/template.hbs

```diff
@@ -1,12 +1,11 @@
 <div class="file-tree">
     {{#if onClickFile}}
-        <a {{action onClickFile root}}>{{root.name}}</a>
+        <a {{action onClickFile root}} role="link">{{root.name}}</a>
```

Comment:
Similar to above: links for navigation, button for interactive elements that keep you on the same page.

(we may not use this widget for much, so not a super high priority. But people tend to code by example, so there's that)